app/workers/sync_extension_contents_at_versions_worker.rb

```diff
@@ -203,8 +203,8 @@ def extract_readme_file_extension(filename)
   end
 
   def ensure_updated_version(readme_body, readme_ext)
-    yml_line_count = @run.cmd("find . -name '*.yml' -o -name '*.yaml' -print0 | xargs -0 wc -l").split("\n").last || ""
-    rb_line_count = @run.cmd("find . -name '*.rb' -print0 | xargs -0 wc -l").split("\n").last || ""
+    yml_line_count = @run.cmd("find . -name '*.yml' -o -name '*.yaml' -print0 | xargs -0 wc -l")&.split("\n")&.last || ""
+    rb_line_count = @run.cmd("find . -name '*.rb' -print0 | xargs -0 wc -l")&.split("\n")&.last || ""
```

Comment:
... because my editor went and fixed up unnecessary whitespace, the fix can be found here.

Reply:
TIL http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/

Reply:
WUT?! 

That would have saved me so much time! 🤦🏾‍♂️